### DISCUSSION TITLE
Multi-user Unable to get independent RSS URL

### DISCUSSION THREAD

**Discussion by socekin** [FreshRSS/FreshRSS#6646] [Q&A]
### Describe the bug

After creating multi-user, when accessing RSS via the public URL, data between users cannot be distinguished.

username1: test1
username2: test2

http://url/i/?a=rss&user=test1
http://url/i/?a=rss&user=test2

These two URLs show the same data.

How should I access RSS between different users?

### To Reproduce

1, Log in as the default user "test"
2, Create a new user "test2"
3, Copy the link, Allows access to all RSS output and refreshing of subscriptions without validation.

### FreshRSS version

1.24.0

### Environment information

- The PHP version is 8.2.18

**Comment by Alkarex** [comment]
The format would be `https://freshrss.example.net/?a=rss&user=alice&token=abc123`
However, the recommended is now to use *user queries*:
https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries

**Comment by socekin** [comment]
![image](https://github.com/user-attachments/assets/7f333401-d7f8-468e-9ab6-68b7e19031f9)
where can i find this setting page?
![CleanShot 2024-07-22 at 09 12 53@2x](https://github.com/user-attachments/assets/f5d58467-aa2e-4e10-90be-7517a6d50663)


`https://freshrss.example.net/?a=rss&user=alice&token=abc123
` The access method is logging into the web page to view. I need to obtain RSS addresses of two different users for use on the mobile RSS reader.

**Reply by Alkarex** [reply]
You first need to save a user query as explained in the doc https://freshrss.github.io/FreshRSS/en/users/user_queries.html

Using the token approach (not recommended), make sure you have a token set up in the corresponding user

**Reply by socekin** [reply]
Thanks, I found it. Is there an API to create a Bookmark for the current query, or can it only be done through the web interface?

**Reply by Alkarex** [reply]
No API at the moment
